pcsx2/gui/ConsoleLogger.cpp

```diff
@@ -324,17 +325,16 @@ static const bool ConLogDefaults[] =
 	true,
 	true,
 	false,
-	true,
 	false,
 	false,
 	false,
 	false,
 	false,
+	true,
 #ifndef DISABLE_RECORDING
 	false,
-	false,
-#endif
 	false
```

Comment:
why were these removed?

Reply:
Sorry was experimenting and forgot about that. I'll fix that in a bit